src/app/components/top-menu/sequential-form/sequential-form.component.ts

```diff
@@ -0,0 +1,97 @@
+import {Component, OnInit} from '@angular/core';
+import {UserActionController} from '../../../gte-core/gte/src/Controller/Main/UserActionController';
+import {UserActionControllerService} from '../../../services/user-action-controller/user-action-controller.service';
+import {UiSettingsService} from '../../../services/ui-settings/ui-settings.service';
+import {SolverService} from '../../../services/solver/solver.service';
+import {TreesFileService} from '../../../services/trees-file/trees-file.service';
+import {MAX_NODES_COUNT_FOR_STRATEGIC_FORM} from '../../../gte-core/gte/src/Utils/Constants';
+
+@Component({
+  selector: 'app-sequential-form',
+  templateUrl: './sequential-form.component.html',
+  styleUrls: ['./sequential-form.component.scss']
+})
+export class SequentialFormComponent implements OnInit {
+  userActionController: UserActionController;
+  stratFormScaleCSS: string;
+  efForm : string;
+  variableMaps : string;
+  include_nash : boolean;
+  include_sequential: boolean;
+  restrict_strategy: boolean;
+  restrict_belief: boolean;
+  showEdit: boolean;
+
+  constructor(private uac: UserActionControllerService,
+              private uis: UiSettingsService,
+              private solver: SolverService,
+              private tts: TreesFileService) {
+  }
+
+
+  ngOnInit() {
+    this.uac.userActionController.subscribe((value) => {
+      this.userActionController = value;
+      // this.userActionController.checkCreateStrategicForm();
+    });
+    this.stratFormScaleCSS = 'scale(' + this.uis.stratFormScale + ')';
+    this.createEfForm()
+    this.include_nash = true
```

Comment:
these should be camelCase and initialized in the property declaration.

Reply:
Agreed.